### DISCUSSION TITLE
Union query support?

### DISCUSSION THREAD

**Discussion by alexanderankin** [tortoise/tortoise-orm#1578] [Q&A]
Hi tortoise-orm maintainers and community,

I was wondering if anybody out there was doing union queries - if tortoise supports those. `create table t1(id varchar(32), data text); create table t2(id varchar(32), data text, category int); select id, data from t1 union select id, data from t2;` thanks.

**Comment by nikita-davydov** [comment]
Bump to this question, it's just impossible to do without raw query
